tests/test_metrics.py

```diff
@@ -86,4 +89,8 @@ def test_metrics(farm, runner):
         assert count_forward[ind].value == N_CALLS
         assert farm.counters()[p][ENDPT_NAME_FWD] == N_CALLS
 
+    # Expect only queue wait time for "status", because that's what `/metrics` gets routed to
+    assert len(count_wait_time) == 1
```

Comment:
In this test, `/status` is called twice (see `N_CALLS`), shouldn't that lead to 2 samples of the metric?

Reply:
It's because in this case you get a sample per bucket. Because both observations fall in the same timing bucket, you get one sample, with value 2.